packages/imperative/CHANGELOG.md

```diff
@@ -2,7 +2,19 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed inconsistent behavior with the `ZOWE_SHOW_SECURE_ARGS` environment variable continuing to mask secure properties when it should not. [#2430](https://github.com/zowe/zowe-cli/issues/2430)
+- Enhancement: Added the `Censor` class, consolidating all sensitive data hiding logic into one class. [#2424](https://github.com/zowe/zowe-cli/pull/2424)
+- Enhancement: Added `showSecureArgs` to `EnvironmentalVariableSettings` to allow extenders to determine if they should mask secure values. [#2424](https://github.com/zowe/zowe-cli/pull/2424)
```

Comment:
we should explain what `showSecureArgs` and `EnvironmentalVariableSettings` are. Are they classes, properties, cats?
Comment applies to Lines 11-14 as well.